_episodes/16-parallel.md

```diff
@@ -73,6 +73,16 @@ def inside_circle(total_count):
 ```
 {: .language-python}
 
+> ## Random Number Generation
+> 
+> For simplicity, random numbers are generated using regular functions, if
```

Comment:
I think the wording here needs to be clearer -- "random numbers are generated" refers to the preceding example code, not the general case.

I propose: "In the preceding code, random numbers are conveniently generated using the built-in capabilities of NumPy. In general, random-number generation is difficult to do well, it's easy to accidentally introduce correlations into the generated sequence. If high quality random numbers are needed..."

Reply:
Thanks. Made the modification.